src/Mappings.js

```diff
@@ -1,4 +1,4 @@
-import * as patterns from './patterns';
+import * as patterns from './Patterns';
 import * as models   from './models';
```

Comment:
Looks like `models` should be PascalCase too

Reply:
`models` is a lowercase directory with a default `index` import so no need to pascal case - however I can update the PR to PascalCase the directory names too if you prefer :+1:

Reply:
Oh yeah - don't know what I'm talking about, all good!